libs/detectors/edgetpu/posenet.py

```diff
@@ -56,14 +57,14 @@ def inference(self, resized_rgb_image):
         Returns:
             result: a dictionary contains of [{"id": 0, "bbox": [x1, y1, x2, y2], "score":s%}, {...}, {...}, ...]
         """
-        assert resized_rgb_image.shape == (721, 1281, 3)
+        assert resized_rgb_image.shape == (self.h, self.w, 3)  # noqa 
```

Comment:
What does #noqa stands for?

Reply:
`#noqa` is used to notify flake8 that the line should be ignored.

However, in that case is not required. I removed the comment.